XCUITests/NavigationTest.swift

```diff
@@ -345,6 +345,41 @@ class NavigationTest: BaseTestCase {
         waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
     }
 
+	func testLongPressOnAddressBar() {
```

Comment:
The test appears enabled on the Smoketest schema, please disable it from there. It should only run in the regular one. Thanks!